x-pack/test/functional/apps/search_playground/serverless.config.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrConfigProviderContext } from '@kbn/test';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const functionalConfig = await readConfigFile(
+    require.resolve('../../../../test_serverless/functional/test_suites/search/config.ts')
+  );
+  const kbnTestServer = functionalConfig.get('kbnTestServer');
+
+  return {
+    ...functionalConfig.getAll(),
+    testFiles: [require.resolve('.')],
+    kbnTestServer: {
+      ...kbnTestServer,
+      serverArgs: [
+        ...kbnTestServer.serverArgs,
+        // used for connector simulators
+        `--xpack.actions.proxyUrl=http://localhost:6200`,
```

Comment:
Any server args overrides for both ES and Kibana won't have affect when these tests are run against real serverless project on MKI, especially things like `localhost` usually break tests in Cloud.

If you don't plan to run tests from the config against real project, nothing to change here. FYI Kibana CI only emulates serverless, it is critically important to run tests in Cloud against real MKI projects.

If you wish to test both on Kibana CI and Cloud (on MKI we run tests every 3 hours against the laster Kibana image) you need to update your tests to work without server args override (e.g. explore API capabilities to set env)

Reply:
I need the proxyUrl for OpenAISimulator as it uses proxyPort. I don't need to override these props; I just need to have them. If the basic config already has actions.proxyUrl, what will happen in that case? Which value will be used?

Reply:
>If the basic config already has actions.proxyUrl, what will happen in that case? Which value will be used?
I guess it depends on how this feature is configured on Cloud. Is it enabled by default? You can also check with Kibana Core Team as they own kibana-controller

Reply:
But I'm not the only one who used these args, so I hope it will work fine :)